backend/plugins/sales_api/src/modules/sales/utils.ts

```
@@ -26,6 +28,26 @@ import {
+  const resolver = resolvers['Deal'] || {};
+
+  for (const subResolver of Object.keys(resolver)) {
+    try {
+      additionInfo[subResolver] = await resolver[subResolver](
+        item,
+        {},
+        { models, user },
+        { isSubscription: true },
+      );
+    } catch (unused) {
+      continue;
```

Biome (1.9.4)

[error] Unnecessary continue statement

Unsafe fix: Delete the unnecessary continue statement


(lint/correctness/noUnnecessaryContinue)